### PR TITLE
Favicon changed to "XM" brand and re-centered the header on main page. 

### PR DESCRIPTION
**Pull request by audieleon**
Just pulled the "XM" favicon from the support page and made it a transparent .png. Set the config to use that image.

Also fixed the centering in the header that the logo broke.

### CODE REVIEW

**Comment by audieleon** [comment]
@jPomeranz The changes for this are in. Backed out the mistakenly merged in changes and fixed up the resulting miss on the merge I did. Should be good to go for only the header's centering and the favicon now.

File: src/components/header.tsx [hunk 1]
```diff
@@ -14,33 +13,40 @@ const Header = ({ siteTitle, centered }: { siteTitle: string; centered: boolean
...
-            minimal
-            onClick={() => setIsOpen(true)}
-          />
+            className={cn('flex items-baseline py-8 text-center', {
+              'max-w-6xl mx-auto': centered,
+              'px-10': centered,
+            })}
+          >
+          <div
+            className={cn('flex items-baseline py-8', {
```
Comment (jPomeranz) [inline src/components/header.tsx:22]: Why is this here a second time?

Comment (jPomeranz) [reply src/components/header.tsx:22]: Undid this as it was breaking the header spacing.
